### PR TITLE
Windows

### PR DESCRIPTION
**Pull request by bodgit**
## Pull Request Checklist

No.

#### General

- [x] Update Changelog following the conventions laid out [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)

- [ ] Update README with any necessary configuration snippets

- [ ] Binstubs are created if needed

- [x] RuboCop passes

- [ ] Existing tests pass

#### New Plugins

- [ ] Tests

- [ ] Add the plugin to the README

- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

#### Purpose

If you try and use these plugins on Windows, you will have to explicitly set the paths as the defaults are hardcoded with Unix-style paths. This PR makes the defaults correct when run on Windows.

### CODE REVIEW

**Comment by bodgit** [comment]
@majormoses could you take a look please?

**Review by majormoses — APPROVED** [review]
LGTM

~Missing [changelog](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md) but its clear enough I can add one for you  here.~

EDIT: somehow I missed the changelog entry...

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
 ## [Unreleased]
+### Changed
```
Comment (majormoses) [inline CHANGELOG.md:7]: Added since you added support.

**Comment by majormoses** [comment]
I manually pushed the gem out since we saw this error: https://travis-ci.org/sensu-plugins/sensu-plugins-puppet/jobs/626473122#L239 I can work on a PR when I have some free time.

**Pull request merged**